rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -177,11 +177,11 @@ def test_exec(config):
             srv_restarted = rgw_service.restart()
 
         if config.test_aync_data_notifications:
+            set_log = "ceph config set global log_to_file true"
```

Comment:
Can we include logic here to avoid running this commands in 4.x??

Reply:
@anrao19, This test would not be added in 4.x.metadata in cephci. It should be added from 5.2 onwards.